src/main/scala/io/joern/scanners/c/UseAfterFree.scala

```diff
@@ -52,4 +53,64 @@ object UseAfterFree extends QueryBundle {
     docFileName = sourcecode.FileName()
   )
 
+  @q
+  def freeReturnedValue()(implicit context: EngineContext): Query = Query(
+    name = "free-returned-value",
+    author = Crew.malte,
+    title = "A value that is returned through a parameter is free'd in a path",
+    description =
+      """
+        |The function sets a field of a function parameter to a value of a local
+        |variable.
+        |This variable is then freed in some paths. Unless the value set in the
+        |function |parameter is overridden later on, the caller has access to the
+        |free'd memory, which is undefined behavior.
+        |
+        |Finds bugs like CVE-2019-18902.
+        |""".stripMargin,
+    score = 5.0,
+    traversal = { cpg =>
+      def outParams =
+        cpg.parameter
+          .typeFullName(".+\\*")
+          .whereNot(
+            _.referencingIdentifiers
+              .argumentIndex(1)
+              .inCall
+              .nameExact("<operator>.assignment", "<operator>.addressOf"))
```

Comment:
you can use the _Operators_ constants here. As in _Operators.assignment_, _Operators.addressOf_, etc.

Reply:
Ah, nice. Somehow I convinced myself these only existed in ocular.